apps/api/src/export/export.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@nestjs/common'
+import { PrismaService } from '../prisma/prisma.service'
+import { Response } from 'express'
+import { createWorkbook } from './helpers/createWorkbook'
+import { ExportableData } from '@prisma/client'
```

Comment:
@AndreyGoranov Can you help me understand what is `ExportableData` as I can find an update of `schema.prisma` and this type is not being generated.

Reply:
It's just a Type:  export Type ExportableData = Donations[] | Campaigns[]